PWGJE/EMCALJetTasks/UserTasks/AliPieceWisePoly.cxx

```diff
@@ -149,7 +150,10 @@ double AliPieceWisePoly::Eval (double x, double* p) {
 }
 
 void AliPieceWisePoly::ReadFSParameters(TString parameterFile, TF1** effFunctions) {
-  TFile* f = new TFile(parameterFile.Data(), "READ");
+  if(parameterFile.Contains("alien://") && ! gGrid)
+    TGrid::Connect("alien://");
+  
+  TFile* f = TFile::Open(parameterFile.Data(), "READ");
```

Comment:
JAliEn-ROOT I think requires

```
TGrid::Connect("alien");
```
What you have (`TGrid::Connect("alien://")`) I think only worked for legacy AliEn

Reply:
Just as an fyi, it seems to still work in the embedding helper: https://github.com/alisw/AliPhysics/blob/0f0caeafbd1cea78059270550ca5a3feab7f3f24/PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcalEmbeddingHelper.cxx#L68

Reply:
I took it from AliTPCPIDResponse where it works with it, but I can change it.